wordpress-mobile/WordPress-iOS
# Fix visual feedback on tappable area

#### This PR adds visual feedback when user tap on parts of the tappable area that are outside of the gravatar image

Fixes # NA

To test:

- build/run or download the installable build
- go to `My Site`
- tap the `Me` button and verify that the tappable area is comfortable and provides visual feedback, regardless the tap occurs inside or outside the gravatar image

PR submission checklist:
